zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```
@@ -898,6 +898,42 @@ paths:
             application/problem+json:
               schema:
                 $ref: "#/components/schemas/ProblemDetail"
+
+  /messages/pubblication:
+    post:
+      tags:
+        - Message
+      summary: Publish a message
+      description: |
+        Publishes a single message. Messages are published to specific partitions computed from
+        their correlation keys.
+        The endpoint does not wait for a correlation result.
+        Use the message correlation endpoint for such use cases.
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/MessagePublicationRequest"
+      responses:
+        "200":
+          description: The message was published.
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/MessagePublicationResponse"
+        "400":
+          description: The provided data is not valid.
+          content:
+            application/problem+json:
+              schema:
+                $ref: "#/components/schemas/ProblemDetail"
+        "500":
+          description: Internal Server Error.
```

```suggestion
          description: Internal server error.
```

:x: Case consistency.